src/api/record/record.ts

```diff
@@ -99,12 +99,22 @@ export class RecordHandler extends RupertPlugin {
           // Update all media links
           record.updateMedia(replaceId);
           this.database[record.id] = record;
+          writeFile(
+            this.getRecordDBPath(record),
+            JSON.stringify(record.toJSON())
+          );
           s.status(200).send(record);
           n();
         });
         return;
       } else {
         this.database[record.id] = record;
+        mkdirp(join(this.dataPath, record.id), () => {
+          writeFile(
+            this.getRecordDBPath(record),
+            JSON.stringify(record.toJSON())
```

Comment:
Can we change line 115 to `JSON.stringify(record.toJSON(), null, 4)` to make the file more human readable? Tested [this solution](http://stackoverflow.com/questions/5670752/how-can-i-pretty-print-json-using-node-js) locally and it seems to do the trick.